src/pages/landing/deriv-go/_footer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Text } from 'components/elements/typography'
+import { Flex } from 'components/containers'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+
+const BannerWrapper = styled(Flex)`
+    overflow: hidden;
+    background-color: var(--color-black);
+    padding: 17px 16px;
+    @media ${device.tabletL} {
+        padding: 16px;
+    }
+`
+const MainWrapper = styled(Flex)`
+    /* max-width: 996px; */
+    margin: 0 auto;
+    flex-direction: row;
+    justify-content: flex-start;
+`
+const FooterTxt = styled(Text)`
+    font-size: 14px;
+    color: var(--color-white);
+    line-height: 1.5;
+`
```

Comment:
Same for this. You can use ` type="sub-paragraph"`  and it will set the `font-size` and `line-height` appropriately

Reply:
Amended in the latest commit